docs/en/changes/changes.md

```diff
@@ -7,6 +7,7 @@
 
 #### OAP Server
 
+* Add Echo component ID(5015) language: Golang.
```

Comment:
Please keep the original changes file style, and add this changes to the latest.